Doc/library/os.rst

```diff
@@ -3392,7 +3392,9 @@ written in Python, such as a mail server's external command delivery program.
    subprocesses.
 
 
-.. function:: posix_spawn(path, argv, env, file_actions=None)
+.. function:: posix_spawn(path, argv, env, file_actions=None, /, *, \
```

Comment:
I don't believe this `/` should be here.

Reply:
See #6725.